frontend/src/scenes/session-recordings/player/PlayerMetaLinks.tsx

```diff
@@ -40,21 +40,67 @@ export function PlayerMetaLinks(): JSX.Element {
         })
     }
 
+    const commonProps: Partial<LemonButtonProps> = {
+        fullWidth: false,
+        size: 'small',
+    }
+
+    const showText = false
+
+    // return (
```

Comment:
commented code 😱 

assuming you're planning on bringing this back at some point 😊

Reply:
Yeah this isn't finished yet. The buttons are borked which led me to [this PR](https://github.com/PostHog/posthog/pull/15351) but it might not get merged easily....